airbyte-server/src/main/java/io/airbyte/server/handlers/WebBackendConnectionsHandler.java

```diff
@@ -74,6 +74,7 @@ public class WebBackendConnectionsHandler {
   private final SchedulerHandler schedulerHandler;
   private final OperationsHandler operationsHandler;
   private final EventRunner eventRunner;
+  // todo (cgardens) - this handler should NOT have access to the db. only access via handler.
```

Comment:
Which handler do you mean. I thought that the config repository was the handler over config persistence. Do you mean that we need to use the connectionsHandler instead?